backend/sass/common.blocks/_wallet.scss

```diff
@@ -4,6 +4,166 @@
 .wallet {
 }
 
+$fullscreen-wallet-grey: #f1f1f1;
+$wallet-top-btm-margin: 0.5rem 0 0.5rem 0;
+$kadena-orange: #F3AB3C;
+
+.fullscreen {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 0.2fr 1.6fr;
+  grid-template-areas: "back header header ." ". wrapper wrapper .";
+}
```

Comment:
Sorry, just stumbled upon this. This should really not be here. Either that fullscreen is an element of the wallet block, then it should be named wallet__fullscreen or it is a block of its own. In that case it should have its own file _fullscreen.

Reply:
Ah, I totally missed the file name. The setup screen code should probably have a `_setup` file, since it's the only place we have the fullscreen stuff anyway. I'll add a ticket to move it

Reply:
Sounds good! Thanks! :-)